components/automate-cs-ocid/habitat/default.toml

```diff
@@ -0,0 +1,50 @@
+# NOTE(ssd) 2018-07-25: These two are not exposed in public-facing
+# configuration as changing them would break everything.
+fips_enabled = false
+ip_mode = "ipv4"
+rails_env="production"
+rack_env="production"
+
+[network]
+port=10114
+listen_ip="0.0.0.0"
+
+[tls]
+root_cert_contents = ""
+cert_contents = ""
+key_contents = ""
+
+[log]
+level = "info"
+rotation_max_bytes = 104857600
+rotation_max_files = 10
+rotation_max_megabytes = 100
+max_error_logs_per_second = 1000
+
+
+[sql]
+timeout = 5000
+pool_init_size = 20
+pool_max_size = 50
+pool_queue_max = 20
+pool_queue_timeout = 2000
+user="automate-cs-ocid"
+db_name="automate-cs-ocid"
+
+
+[depsolver]
```

Comment:
Does the OCID service make use of these depsolver configs? I believe they only apply to the erchef service.

Reply:
Good catch @teknofire , We missed to remove this.

Reply:
Resolved it in https://github.com/chef/automate/pull/7909/commits/f1728e03a1b0b95d08b8af199493a3de50f67d7e